src/Microsoft.ML.Dnn/DnnUtils.cs

```diff
@@ -293,23 +287,23 @@ internal static DnnModel LoadDnnModel(IHostEnvironment env, ImageClassificationE
                         ZipFile.ExtractToDirectory(Path.Combine(Directory.GetCurrentDirectory(), @"tfhub_modules.zip"), @"tfhub_modules");
                     }
                 }
-                else if (arch == ImageClassificationEstimator.Architecture.ResnetV2101)
+                else if (arch == Architecture.ResnetV2101)
                 {
                     var baseGitPath = @"https://aka.ms/mlnet-resources/image/ResNet101Tensorflow/resnet_v2_101_299.meta";
                     using (WebClient client = new WebClient())
```

Comment:
Recommend using Yael's ResourceManager. 

Example from the WordEmbedding transform:
https://github.com/dotnet/machinelearning/blob/5163413f3cfab430e49f6fb2a690c160fde46b58/src/Microsoft.ML.Transforms/Text/WordEmbeddingsExtractor.cs#L631-L642

Reply:
removed this change as part of PR but good recommendation will add that later.

---
In reply to: [338999752](https://github.com/dotnet/machinelearning/pull/4384#discussion_r338999752) [](ancestors = 338999752)